pkg/util/validation/exporter/exporter_test.go

```diff
@@ -154,7 +155,20 @@ func TestOverridesExporterWithRing(t *testing.T) {
 		return desc, true, nil
 	}))
 
-	time.Sleep(100 * time.Millisecond)
+	instanceAddressesFromReplicationSet := func(set ring.ReplicationSet) []string {
```

Comment:
Just call `set.GetAddresses()`.

Reply:
🤦‍♂️ that sounds like a pretty good idea :)